src/utils/getAddressesForMain.ts

```diff
@@ -0,0 +1,6 @@
+import { ConfigHelper } from '@oceanprotocol/lib'
```

Comment:
do we need this utility?

Reply:
no, i needed it before the last update in the graph, i had to filter the tokens, now the ocean reserve is on the transaction. Will remove it